firefox_puppeteer/tests/test_windows.py

```diff
@@ -128,9 +128,16 @@ def closer(window):
         win2 = BaseWindow(lambda: self.marionette, win2.handle)
 
         self.assertEquals(len(self.marionette.chrome_window_handles), 2)
-
         win2.close(callback=closer)
-        win1.switch_to()
+
+        win1.focus()
+
+        # We cannot open a chrome window in the background yet, so this will fail
```

Comment:
I wanted to with the initial implementation but it was too hard and I wasn't able to get it working. Given that we don't have this feature yet, we should not mention it here. I will update the comment.